akka-stream/src/main/scala/akka/stream/javadsl/Source.scala

```diff
@@ -307,6 +306,53 @@ object Source {
   def queue[T](bufferSize: Int, overflowStrategy: OverflowStrategy): Source[T, SourceQueue[T]] =
     new Source(scaladsl.Source.queue[T](bufferSize, overflowStrategy).mapMaterializedValue(new SourceQueueAdapter(_)))
 
+  /**
+    * Creates Source that works as dispatcher for legacy blocking IO API
```

Comment:
"Creates a `Source` from a blocking API" (if it is Legacy or IO we don't really know here, right?)